app/javascript/src/navigation/components/Header.js

```diff
@@ -1,71 +1,176 @@
 
 export class Header extends Component {
   // lifecycle
   render () {
-    return <div {...rules.container}>
-      <Link {...rules.logoLink} to='/'>
-        <img src={logo} alt='Legislated' height='40' width='40' />
-        <span>LEGISLATED</span>
-      </Link>
-      <MobileNav />
-      <div {...rules.nav}>
-        <NavLinkList />
+    return (
+      <div>
+        <DesktopHeader>
+          <HeroLogo>
+            <LogoLink to='/'>
+              <img src={logo} alt='Legislated' height='64' width='64' />
+            </LogoLink>
+            <LogoLink to='/'>
+              <span>Legislated</span>
+            </LogoLink>
+          </HeroLogo>
+          <Sticky topOffset={280}>
+            {({ isSticky, style }) => {
+              return (
+                <HeaderNav isSticky={isSticky} style={style}>
+                  {isSticky && <StickyLogo>
+                    <LogoLink to='/'>
+                      <img src={logo} alt='Legislated' height='40' width='40' />
+                    </LogoLink>
+                    <LogoLink to='/'>
+                      <span>LEGISLATED</span>
+                    </LogoLink>
+                  </StickyLogo>}
+                  <HeaderNavCenter>
+                    <NavLink to='/' exact>Home</NavLink>
+                    <NavLink to='/faq' exact>FAQ</NavLink>
+                    <NavLink to='/about' exact>About</NavLink>
+                  </HeaderNavCenter>
+                </HeaderNav>
+              )
+            }}
+          </Sticky>
+        </DesktopHeader>
+        <MobileHeader>
+          <div>
+            <LogoLink to='/'>
+              <img src={logo} alt='Legislated' height='40' width='40' />
+            </LogoLink>
+            <LogoLink to='/'>
+              <span>LEGISLATED</span>
+            </LogoLink>
+          </div>
+          <MobileNav />
+        </MobileHeader>
       </div>
-    </div>
+    )
   }
 }
 
-const rules = stylesheet({
-  container: {
-    ...mixins.borders.low(['bottom']),
-    display: 'flex',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    height: 80,
-    paddingLeft: 30,
-    paddingRight: 30,
-    backgroundColor: alpha(colors.background, 0.95),
-    ...mixins.mobile({
-      height: 60,
-      paddingLeft: 15,
-      paddingRight: 15
-    })
-  },
-  logoLink: {
-    color: colors.black,
-    fontSize: 32,
-    textDecoration: 'none',
-    transition: 'color 0.25s',
-    ':hover': {
-      color: alpha(colors.black, 0.6)
-    },
-    '> img': {
-      verticalAlign: 'top'
-    },
-    '> span:last-child': {
-      ...mixins.fonts.bold,
-      marginLeft: 15,
-      letterSpacing: 5
-    },
-    ...mixins.mobile({
-      fontSize: 28
-    })
-  },
-  nav: {
-    ...mixins.borders.low(['left']),
-    flex: 1,
-    display: 'flex',
-    marginLeft: 15,
-    paddingLeft: 15,
-    ...mixins.mobile({
-      display: 'none'
-    })
+const DesktopHeader = styled.div`
+  @media (max-width: 700px) {
```

Comment:
can we use a mixin to keep the breakpoints consistent?

Reply:
yes, its a little verbose emulating the current mixin setup, something like:
```
function mobile(styles) {
  return css`
    @media (max-width: 700px) {
      ${styles}
    }
  `
}
```
I held off on investing too much in creating idiomatic/shared `emotion` stuff until we decided to use it going forward.

Reply:
yeah, makes sense to hold off for now. could probably cut down some of the verbosity of the first option using an arrow function:

```js
const mobile = (styles) => css`
  @media (max-width: 700px) {
    ${stringStyles}
  }
`
```